### DISCUSSION TITLE
How can I display the time on the corner of each timeslot?

### DISCUSSION THREAD

**Discussion by stescacom** [vkurko/calendar#246] [Q&A]
I want to display the time value in the slots how can I achieve this? Is there a label option for it?

<img width="1041" alt="image" src="https://github.com/vkurko/calendar/assets/93092835/2c14ddd5-b4de-44eb-86b4-b8fc3cc267b5">

**Comment by vkurko** [comment]
Unfortunately, there is no such option. A solution would be to generate background events for each slot and override how they are rendered using [eventContent](https://github.com/vkurko/calendar#eventcontent).

**Reply by t-lock** [reply]
@vkurko How can I show the time when hovering a slot, like in your demo? 
![image](https://github.com/vkurko/calendar/assets/11277885/67b899d3-2d5e-4ed5-9da5-810769a6dbf1)

I am not seeing anything in the options in the source code, or documentation

**Reply by vkurko** [reply]
There is [pointer](https://github.com/vkurko/calendar?tab=readme-ov-file#pointer) option for this.
